dashboard/src/assets/constants/overviewConstants.js

```diff
@@ -18,6 +18,12 @@ export const IS_DIRTY_SERVER_DELETE = "isDirtyDate";
 export const IS_EDIT = "isEdit";
 export const IS_ITEM_FAVORITED = "isItemFavorited";
 export const IS_ITEM_SEEN = "isItemSeen";
+export const KEYS_JOIN_BY = "*";
+export const KEY_INDEX_REGEX = /^server[.*]index-map$/;
+export const KEY_ITERATIONS_REGEX = /^dataset[.*]metalog[.*]iterations/;
+export const KEY_OPERATIONS_REGEX = /^dataset[.*]operations$/;
+export const KEY_TOOLS_REGEX = /^dataset[.*]metalog[.*]tools/;
+export const KEY_TARBALL_PATH_REGEX = /^server[.*]tarball-path$/;
```

Comment:
> I don't like the wildcarded separator pattern... _and_ the `[]` grouping is meaningless here since there's a single character inside.

@dbutenhof, I think you might be confused.  I think the `[]` means "match any (one) of these (literal) characters", so we're matching on the keys separated by either a _period_ **or** an _asterisk_ -- there's no wildcard here (other than the alternation), and there are _two_ characters in the set.

(Does that change any of your other comments, here?)